config/logging_config_example.py

```diff
@@ -0,0 +1 @@
+LOGFILE = 'path/fatal_errors.log'
```

Comment:
Logging level should be also placed in config. In order to not doing imports in config file, you can specify level here as simple constant, and convert it to object from `logging` using enumeration, for example. 

Do not forget to check is this constant is specified correctly and match one of log levels, using in our project.

INFO should be level by default.